cloud/google/start_etcd.sh

```diff
@@ -26,7 +26,12 @@ done
 wait
 
 MANIFEST=/tmp/etcd_container.yaml
-DISCOVERY=$(curl -s https://discovery.etcd.io/new?size=3)
+echo -n "Getting Discovery URL"
+while [ "${DISCOVERY}" == "" ]; do
+  DISCOVERY=$(curl -s https://discovery.etcd.io/new?size=${ETCD_NUM_REPLICAS})
```

Comment:
In the way we're doing it, yes.
There are other methods, but this one's simplest and works.